### PR TITLE
Rc 3170

### PR DESCRIPTION
**Pull request by prckent**
## Proposed changes

Proposed v3.17.0 release.

Please check CHANGELOG.md for notes on release and CMakeLists.txt.

~I will post a full list of PRs for review~

## What type(s) of changes does this code introduce?

- Bugfix
- New feature
- Code style update (formatting, renaming)
- Refactoring (no functional changes, no api changes)
- Build related changes
- Testing changes (e.g. new unit/integration/performance tests)
- Documentation changes

### Does this introduce a breaking change?

- Yes

## What systems has this change been tested on?

No significant changes since last weekend runs.

## Checklist

- Yes. This PR is up to date with current the current state of 'develop'
- Mostly. Code added or changed in the PR has been clang-formatted
- Yes. This PR adds tests to cover any new code, or to catch a bug that is being fixed
- Yes. Documentation has been added (if appropriate)

### CODE REVIEW

**Comment by prckent** [comment]
List of PRs since v3.16.0:

*  add mw_ APIs to RotatedSPO [#4701](https://github.com/QMCPACK/qmcpack/pull/4701)
*  Fix Nexus GPU jobs for NERSC/Perlmutter [#4699](https://github.com/QMCPACK/qmcpack/pull/4699)
*  Add NERSC perlmutter recipe. [#4698](https://github.com/QMCPACK/qmcpack/pull/4698)
*  Move nan check from DriftModifierUNR to TWF [#4697](https://github.com/QMCPACK/qmcpack/pull/4697)
*  pin clang and gcc versions for ubuntu22-openmpi [#4696](https://github.com/QMCPACK/qmcpack/pull/4696)
*  Add evaluateDerivativesWF for J3 [#4694](https://github.com/QMCPACK/qmcpack/pull/4694)
*  Improve cublas getrf/getri_batched error handling. [#4693](https://github.com/QMCPACK/qmcpack/pull/4693)
*  Update device manager [#4692](https://github.com/QMCPACK/qmcpack/pull/4692)
*  Guard batch size computation against zero [#4691](https://github.com/QMCPACK/qmcpack/pull/4691)
*  Add tests for orbital rotation for Be [#4689](https://github.com/QMCPACK/qmcpack/pull/4689)
*  Free MKL reserved memory before the SYCL queue destruction. [#4687](https://github.com/QMCPACK/qmcpack/pull/4687)
*  Correct app_error usage [#4686](https://github.com/QMCPACK/qmcpack/pull/4686)
*  Remove xxx_unit cmake targets [#4682](https://github.com/QMCPACK/qmcpack/pull/4682)
*  Fix error message printing issue in the safeguard exception catch [#4680](https://github.com/QMCPACK/qmcpack/pull/4680)
*  Change std::isnan to qmcplusplus::isnan [#4679](https://github.com/QMCPACK/qmcpack/pull/4679)
*  Make Intel LLVM compiler to pass qmcplusplus::isnan tests. [#4678](https://github.com/QMCPACK/qmcpack/pull/4678)
*  Introduce qmcplusplus::isnan [#4677](https://github.com/QMCPACK/qmcpack/pull/4677)
*  Introduce QMC_DISABLE_HIP_HOST_REGISTER. ON by default. [#4674](https://github.com/QMCPACK/qmcpack/pull/4674)
*  RandomBase: use type alias FullPrecRealType instead of double. [#4672](https://github.com/QMCPACK/qmcpack/pull/4672)
*  Nexus: Use simplices in convex_hull [#4671](https://github.com/QMCPACK/qmcpack/pull/4671)
*  Update references for positron deterministic test [#4669](https://github.com/QMCPACK/qmcpack/pull/4669)
*  Make LCAOrbitalSet a member of LCAOrbitalSetWithCorrection [#4668](https://github.com/QMCPACK/qmcpack/pull/4668)
*  transform multidet signed to unsigned int [#4667](https://github.com/QMCPACK/qmcpack/pull/4667)
*  Adjust CuspCorrection [#4665](https://github.com/QMCPACK/qmcpack/pull/4665)
*  Use SYCL in-order queue. [#4663](https://github.com/QMCPACK/qmcpack/pull/4663)
*  Fix calling path_.back() on an empty string [#4662](https://github.com/QMCPACK/qmcpack/pull/4662)
*  Use base class instead of templates [#4659](https://github.com/QMCPACK/qmcpack/pull/4659)
*  Improve some assignment operators [#4658](https://github.com/QMCPACK/qmcpack/pull/4658)
*  use int64_t when aggregating accept/reject counts. [#4657](https://github.com/QMCPACK/qmcpack/pull/4657)
*  AFQMC: Use a reference instead of a pointer. [#4656](https://github.com/QMCPACK/qmcpack/pull/4656)
*  Nexus: add pdos flag for Projwfc [#4655](https://github.com/QMCPACK/qmcpack/pull/4655)
*  Nexus: Adding crowds_serialize_walkers tag to dmc input list [#4651](https://github.com/QMCPACK/qmcpack/pull/4651)
*  Create RandomBase class and FakeRandom and StdRandom classes inherit from it [#4650](https://github.com/QMCPACK/qmcpack/pull/4650)
*  RandomGenerator: Avoid extra construction and memory allocations [#4649](https://github.com/QMCPACK/qmcpack/pull/4649)
*  Simplify checkpointing, change defaults, fix legacy, implement in batched [#4646](https://github.com/QMCPACK/qmcpack/pull/4646)
*  Nexus: update qdens to handle batched driver input/output [#4645](https://github.com/QMCPACK/qmcpack/pull/4645)
*  Nexus: fix namelist read for Projwfc input [#4644](https://github.com/QMCPACK/qmcpack/pull/4644)
*  Removing debug output for InputSection [#4642](https://github.com/QMCPACK/qmcpack/pull/4642)
*  Fix MSD when there is only 1 det in a given spin channel. [#4640](https://github.com/QMCPACK/qmcpack/pull/4640)
*  Allow the use of "cell" and "corner" parameters in spindensity qdens analysis [#4638](https://github.com/QMCPACK/qmcpack/pull/4638)
*  Fix segfault with orbital rotation and multiple determinants [#4635](https://github.com/QMCPACK/qmcpack/pull/4635)
*  Remove performance test input files for legacy cuda. [#4632](https://github.com/QMCPACK/qmcpack/pull/4632)
*  Handle the special case of empty spin channels in Nexus [#4631](https://github.com/QMCPACK/qmcpack/pull/4631)
*  Update NiO h5 download link. [#4629](https://github.com/QMCPACK/qmcpack/pull/4629)
*  Fix sposet_indices missing construction in OrbitalImages. [#4627](https://github.com/QMCPACK/qmcpack/pull/4627)
*  Fix Be orbital optimization test [#4626](https://github.com/QMCPACK/qmcpack/pull/4626)
*  Cleanup snprintf calls with lessons learned [#4622](https://github.com/QMCPACK/qmcpack/pull/4622)
*  Add statistical integraton and deterministic test for positron [#4621](https://github.com/QMCPACK/qmcpack/pull/4621)
*  Update Catch2 to v2.13.9. Remove sprintf [#4620](https://github.com/QMCPACK/qmcpack/pull/4620)
*  Orbital rotation optimization test for Be atom [#4619](https://github.com/QMCPACK/qmcpack/pull/4619)
*  Remove remaining instances of sprintf outside of external_codes [#4618](https://github.com/QMCPACK/qmcpack/pull/4618)
*  Remove unused functions with sprintf [#4616](https://github.com/QMCPACK/qmcpack/pull/4616)
*  Create TimerList class [#4614](https://github.com/QMCPACK/qmcpack/pull/4614)
*  Use std::snprintf in QMCCostFunctionBase. [#4613](https://github.com/QMCPACK/qmcpack/pull/4613)
*  J1Spin indexing issue [#4612](https://github.com/QMCPACK/qmcpack/pull/4612)
*  Add LCAO mw_evaluateValue and mw_evaluateDetRatios [#4611](https://github.com/QMCPACK/qmcpack/pull/4611)
*  Add Be STO VMC test [#4609](https://github.com/QMCPACK/qmcpack/pull/4609)
*  switch from sprintf to std::snprintf [#4608](https://github.com/QMCPACK/qmcpack/pull/4608)
*  Use snprintf or std::cout instead of sprintf. [#4607](https://github.com/QMCPACK/qmcpack/pull/4607)
*  Icpx misc code and recipe update [#4606](https://github.com/QMCPACK/qmcpack/pull/4606)
*  migrate from dockerhub to the github container registry  [#4605](https://github.com/QMCPACK/qmcpack/pull/4605)
*  Fix Nexus rtd configuration [#4604](https://github.com/QMCPACK/qmcpack/pull/4604)
*  Introduce ENABLE_PPCONVERT option and skip ppconvert compilation when cross compiling. [#4601](https://github.com/QMCPACK/qmcpack/pull/4601)
*  Clarify the behavior of the matrix_inverter option. [#4600](https://github.com/QMCPACK/qmcpack/pull/4600)
*  Automatically apply additional clang offload optimization flags. [#4599](https://github.com/QMCPACK/qmcpack/pull/4599)
*  Nexus support for Kagayaki cluster at JAIST [#4598](https://github.com/QMCPACK/qmcpack/pull/4598)
*  Fixing multideterminant HDF5 file to unsigned 64 bit integers [#4597](https://github.com/QMCPACK/qmcpack/pull/4597)
*  Replace set-output in GitHub Actions [#4596](https://github.com/QMCPACK/qmcpack/pull/4596)
*  Reduce restart_batch memory consumption when running 8r-2t. [#4594](https://github.com/QMCPACK/qmcpack/pull/4594)
*  update runtime env to fix read the docs [#4592](https://github.com/QMCPACK/qmcpack/pull/4592)
*  Relax test_ecp tolerance. [#4589](https://github.com/QMCPACK/qmcpack/pull/4589)
*  Update hdf_archive push [#4585](https://github.com/QMCPACK/qmcpack/pull/4585)
*  Make the position specs in tests more compact, part 2 [#4584](https://github.com/QMCPACK/qmcpack/pull/4584)
*  Handle disabled variational parameters in rotation derivatives [#4581](https://github.com/QMCPACK/qmcpack/pull/4581)
*  Read and write orbital rotation parameters [#4580](https://github.com/QMCPACK/qmcpack/pull/4580)
*  update ci ubuntu images from 20.04 to 22.04 [#4579](https://github.com/QMCPACK/qmcpack/pull/4579)
*  Update nightly scripts [#4578](https://github.com/QMCPACK/qmcpack/pull/4578)
*  Fix Typos in C Diamond 1x1x1 Complex Twist Wave Function File for Test [#4575](https://github.com/QMCPACK/qmcpack/pull/4575)
*  Update codecov configuration to notify after 2 builds [#4574](https://github.com/QMCPACK/qmcpack/pull/4574)
*  Introduce -fdisable-host-devmem option for upcoming AOMP/ROCm compilers [#4572](https://github.com/QMCPACK/qmcpack/pull/4572)
*  Refactor ornl sulfur and nitrogen CI [#4571](https://github.com/QMCPACK/qmcpack/pull/4571)
*  Fix defective CI script [#4569](https://github.com/QMCPACK/qmcpack/pull/4569)
*  Fix 1RDM estimator data written to stat.h5 [#4568](https://github.com/QMCPACK/qmcpack/pull/4568)
*  Add chmod +x permissions to CI scripts [#4567](https://github.com/QMCPACK/qmcpack/pull/4567)
*  Update sulfur CI scripts [#4562](https://github.com/QMCPACK/qmcpack/pull/4562)
*  Make the position specification in tests more compact [#4559](https://github.com/QMCPACK/qmcpack/pull/4559)
*  Nexus: use data hashing in heuristic equilibration detection algorithm [#4557](https://github.com/QMCPACK/qmcpack/pull/4557)
*  add correct indexing for SpinDensityNew output [#4552](https://github.com/QMCPACK/qmcpack/pull/4552)
*  Fix frontier build recipe when make install [#4551](https://github.com/QMCPACK/qmcpack/pull/4551)
*  Frontier recipe and job script [#4550](https://github.com/QMCPACK/qmcpack/pull/4550)
*  Change coulomb safeguard printout to uniform error [#4548](https://github.com/QMCPACK/qmcpack/pull/4548)
*  skip ppconvert compilation when isnan unsupported [#4545](https://github.com/QMCPACK/qmcpack/pull/4545)
*  Read and write variational parameters [#4541](https://github.com/QMCPACK/qmcpack/pull/4541)
*  Add shared resource in LCAO [#4538](https://github.com/QMCPACK/qmcpack/pull/4538)
*  Update resource handling [#4537](https://github.com/QMCPACK/qmcpack/pull/4537)
*  Stick to CUDA 11.8 in CI [#4535](https://github.com/QMCPACK/qmcpack/pull/4535)
*  Split restart_dmc into two tests with and without disable_branching. [#4534](https://github.com/QMCPACK/qmcpack/pull/4534)
*  Magnetization Density Estimator [#4531](https://github.com/QMCPACK/qmcpack/pull/4531)
*  fix code hightlighting default in devloping section. upd copyright [#4530](https://github.com/QMCPACK/qmcpack/pull/4530)
*  Remove unnecessary hdf5 error checking [#4529](https://github.com/QMCPACK/qmcpack/pull/4529)
*  Add restart_dmc test with nontrivial walker weights. [#4526](https://github.com/QMCPACK/qmcpack/pull/4526)
*  Code cleanup from work on but unrelated to #4493. [#4524](https://github.com/QMCPACK/qmcpack/pull/4524)
*  Pass RuntimeOptions to TrialWaveFunction [#4523](https://github.com/QMCPACK/qmcpack/pull/4523)
*  bugfix for MSD with spins and add mw_ tests [#4522](https://github.com/QMCPACK/qmcpack/pull/4522)
*  Additional updates for InputSection consistency [#4521](https://github.com/QMCPACK/qmcpack/pull/4521)
*  Remove using namespace std from header file [#4520](https://github.com/QMCPACK/qmcpack/pull/4520)
*  add centos to listed docker files in docs [#4519](https://github.com/QMCPACK/qmcpack/pull/4519)
*  qmc-fit: Added equation of states and morse fits with jackknife [#4518](https://github.com/QMCPACK/qmcpack/pull/4518)
*  Protect the available thread count against library calls changing it [#4515](https://github.com/QMCPACK/qmcpack/pull/4515)
*  remove unused path from centos ci job for clarity [#4514](https://github.com/QMCPACK/qmcpack/pull/4514)
*  Detect amdgpu using gfx instead amdgcn. [#4511](https://github.com/QMCPACK/qmcpack/pull/4511)
*  Update ALCF Theta recipe to build with gcc. [#4509](https://github.com/QMCPACK/qmcpack/pull/4509)
*  Simplify SplineR2R rotation [#4508](https://github.com/QMCPACK/qmcpack/pull/4508)
*  Introduce QMC_GPU_ARCHS option [#4507](https://github.com/QMCPACK/qmcpack/pull/4507)
*  Change DMC population control to use population weight [#4506](https://github.com/QMCPACK/qmcpack/pull/4506)
*  Complex Spline rotation api implementation [#4505](https://github.com/QMCPACK/qmcpack/pull/4505)
*  mw_ APIs for SOECPotential and SOECPComponent [#4504](https://github.com/QMCPACK/qmcpack/pull/4504)
*  Fix documentation [#4503](https://github.com/QMCPACK/qmcpack/pull/4503)
*  Update nitrogen CI script permission [#4500](https://github.com/QMCPACK/qmcpack/pull/4500)
*  Complete remaining QMC_CUDA related tasks [#4499](https://github.com/QMCPACK/qmcpack/pull/4499)
*  Remove raw owning pointers in SampleStack [#4496](https://github.com/QMCPACK/qmcpack/pull/4496)
*  Reinstate ornl-nitrogen-1 CI ROCm jobs [#4495](https://github.com/QMCPACK/qmcpack/pull/4495)
*  Housekeeping around walker count handling [#4494](https://github.com/QMCPACK/qmcpack/pull/4494)
*  Restrict Eref update history [#4492](https://github.com/QMCPACK/qmcpack/pull/4492)
*  Remove the error trap which blocks DMC->any QMC restart in classic drivers [#4491](https://github.com/QMCPACK/qmcpack/pull/4491)
*  Fix units in qdens-radial and add documentation [#4490](https://github.com/QMCPACK/qmcpack/pull/4490)
*  Configure performance test to include full path [#4489](https://github.com/QMCPACK/qmcpack/pull/4489)
*  Document how the trial energy is updated. [#4488](https://github.com/QMCPACK/qmcpack/pull/4488)
*  Error abort for unsupported non-DMC restarts after DMC in legacy drivers [#4486](https://github.com/QMCPACK/qmcpack/pull/4486)
*  Sphinx-related doc fixes [#4485](https://github.com/QMCPACK/qmcpack/pull/4485)
*  Fix/improve walker population carry over handling in batched drivers [#4484](https://github.com/QMCPACK/qmcpack/pull/4484)
*  Fix walker initialization in batched drivers when the request number is smaller than the carry-over [#4482](https://github.com/QMCPACK/qmcpack/pull/4482)
*  Save walker weights in HDF5 files [#4481](https://github.com/QMCPACK/qmcpack/pull/4481)
*  Add ProjectData ref member to WaveFunctionPool from QMCMain [#4480](https://github.com/QMCPACK/qmcpack/pull/4480)
*  docs(): correct the wrong .dat filename in the afqmc tutorial [#4478](https://github.com/QMCPACK/qmcpack/pull/4478)
*  Add Pathak-Wagner Regularizer to ACForce [#4477](https://github.com/QMCPACK/qmcpack/pull/4477)
*  qmc-fit: Add parameter fitting with jackknife for e.g. DFT+U, EXX scans [#4475](https://github.com/QMCPACK/qmcpack/pull/4475)
*  Add Magnetization Density Input Class [#4474](https://github.com/QMCPACK/qmcpack/pull/4474)
*  Make lattice initialization compact in unit tests [#4473](https://github.com/QMCPACK/qmcpack/pull/4473)
*  Remove remnants of the released node method. [#4472](https://github.com/QMCPACK/qmcpack/pull/4472)
*  Cleanup code using walker->Weight [#4471](https://github.com/QMCPACK/qmcpack/pull/4471)
*  Enable orbital rotation [#4470](https://github.com/QMCPACK/qmcpack/pull/4470)
*  Store copy of bspline coefficients for orbital rotation [#4469](https://github.com/QMCPACK/qmcpack/pull/4469)
*  Used shared pointer for LCAO coefficient matrix copy [#4468](https://github.com/QMCPACK/qmcpack/pull/4468)
*  SplineSPO C2R Hybrid rep code path corrected [#4465](https://github.com/QMCPACK/qmcpack/pull/4465)
*  Fix clang warning about non-virtual destructor [#4464](https://github.com/QMCPACK/qmcpack/pull/4464)
*  Add delta rotation functions for orbital rotation [#4463](https://github.com/QMCPACK/qmcpack/pull/4463)
*  Remove QMCMainState and CoupledMC [#4461](https://github.com/QMCPACK/qmcpack/pull/4461)
*  Update deprecated python time.clock with time.process_time [#4460](https://github.com/QMCPACK/qmcpack/pull/4460)
*  Remove unused VMC variables [#4459](https://github.com/QMCPACK/qmcpack/pull/4459)
*  Update bmpi3 [#4458](https://github.com/QMCPACK/qmcpack/pull/4458)
*  Spo unit tests add resource [#4457](https://github.com/QMCPACK/qmcpack/pull/4457)
*  Rename QMCAppBase and QMCMain members and one ifdef QMC_COMPLEX [#4456](https://github.com/QMCPACK/qmcpack/pull/4456)
*  Remove unused alias types [#4455](https://github.com/QMCPACK/qmcpack/pull/4455)
*  Cleanup GridExternalPotential class [#4454](https://github.com/QMCPACK/qmcpack/pull/4454)
*  Cleanup ForwardWalking class [#4453](https://github.com/QMCPACK/qmcpack/pull/4453)
*  Add single thread offload NiO performance tests [#4452](https://github.com/QMCPACK/qmcpack/pull/4452)
*  Cleanup ForceBase and BareForce classes [#4451](https://github.com/QMCPACK/qmcpack/pull/4451)
*  Modify input for Rotated SPOs [#4449](https://github.com/QMCPACK/qmcpack/pull/4449)
*  Make one definition of SPOMap [#4446](https://github.com/QMCPACK/qmcpack/pull/4446)
*  Fix spline block offset caused by #4439 [#4443](https://github.com/QMCPACK/qmcpack/pull/4443)
*  Remove all QMC_CUDA dependencies on testsets [#4442](https://github.com/QMCPACK/qmcpack/pull/4442)
*  Hybridrep mock-up implementation for batched/offload [#4441](https://github.com/QMCPACK/qmcpack/pull/4441)
*  Construct crowd data before each use in the batch optimizer [#4440](https://github.com/QMCPACK/qmcpack/pull/4440)
*  Refactor low level routines used by Spline SPO offload [#4439](https://github.com/QMCPACK/qmcpack/pull/4439)
*  Fix orbital rotation RHF parameter derivatives [#4437](https://github.com/QMCPACK/qmcpack/pull/4437)
*  Remove buildOptVariables from SPOSet [#4435](https://github.com/QMCPACK/qmcpack/pull/4435)
*  Add sposets_for_testing CMake target ConstantSPOSet FakeSPO [#4433](https://github.com/QMCPACK/qmcpack/pull/4433)
*  Remove Legacy CUDA and disable QMC_CUDA build option [#4431](https://github.com/QMCPACK/qmcpack/pull/4431)
*  Create ConstantSPOSet for Unit Testing [#4414](https://github.com/QMCPACK/qmcpack/pull/4414)
*   add docker tests for spack build centos image [#4404](https://github.com/QMCPACK/qmcpack/pull/4404)
*  Initial unit tests to DensityEstimator class [#4386](https://github.com/QMCPACK/qmcpack/pull/4386)
*  Improve determinant list signed integer handling [#4355](https://github.com/QMCPACK/qmcpack/pull/4355)

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,62 @@
...
+* The legacy CUDA implementation, the version built with QMC_CUDA=1, has been removed from the codebase,
+  [#4431](https://github.com/QMCPACK/qmcpack/pull/4431),
+  [#4632](https://github.com/QMCPACK/qmcpack/pull/4632),[#4499](https://github.com/QMCPACK/qmcpack/pull/4499),
+  [#4442](https://github.com/QMCPACK/qmcpack/pull/4442).
+* For increased performance with current AMD GPU support, new QMC_DISABLE_HIP_HOST_REGISTER option is enabled by default for
+  ROCm/HIP builds. [#4674](https://github.com/QMCPACK/qmcpack/pull/4674)
+* Bugfix: J1Spin indexing was wrong [#4612](https://github.com/QMCPACK/qmcpack/pull/4612)
+* Bugfix: 1RDM estimator data written to stat.h5 was incorrect [#4568](https://github.com/QMCPACK/qmcpack/pull/4568)
+* Introduced ENABLE_PPCONVERT option and skip ppconvert compilation when cross compiling. [#4601](https://github.com/QMCPACK/qmcpack/pull/4601)
+* Faster builds compared to v3.16.0 due to code refactoring
```
Comment (quantumsteve) [inline CHANGELOG.md:40]:
```suggestion
* Faster builds compared to v3.16.0 due to code refactoring [#4682](https://github.com/QMCPACK/qmcpack/pull/4682)
```

**Comment by prckent** [comment]
Test this please

**Comment by prckent** [comment]
Test this please

**Comment by prckent** [comment]
Confirming reports at 3.17.0:
```
112:   Input file(s): qmc_short.in.xml 
112: 
112: =====================================================
112:                     QMCPACK 3.17.0
112: 
112:        (c) Copyright 2003-  QMCPACK developers
112: 
112:                     Please cite:
112:  J. Kim et al. J. Phys. Cond. Mat. 30 195901 (2018)
112:       https://doi.org/10.1088/1361-648X/aab9c3
112: 
112:   Git branch: rc_3170
112:   Last git commit: af3ac23570f80d6ce4491864c0eeb1717ee3e8f4
112:   Last git commit date: Fri Aug 18 15:12:53 2023 -0400
112:   Last git commit subject: Add link
112: =====================================================
112:   Global options 
112: 
112:   Total number of MPI ranks = 1
112:   Number of MPI groups      = 1
112:   MPI group ID              = 0
112:   Number of ranks in group  = 1
112:   MPI ranks per node        = 1
112:   OMP 1st level threads     = 16
```

**Review by markdewing — APPROVED** [review]

**Comment by prckent** [comment]
Test failures on the rc are the usual canonical suspects (`ctest -E long` output):
```
The following tests FAILED:
         94 - estimator-acforce-legacy-1-16-check (Failed)
         96 - estimator-acforce-fast-1-16-check (Failed)
        325 - rmg-Diamond2-1x1x1-gamma-ccECP-np-1-1-16-kinetic (Failed)
        430 - short-C4_md-ae-ground-qp-vmc-1-16-variance (Failed)
        440 - short-LiH_ae-vmc_hf_noj_estimator_energydensity_voronoi-4-4-check (Failed)
        534 - short-LiH_pp-vmc_hf_sdj_estimator_spindensity-4-4 (Failed)
        535 - short-LiH_pp-vmc_hf_sdj_estimator_spindensity-4-4-check (Failed)
        537 - short-LiH_pp-vmc_hf_sdj_estimator_energydensity_voronoi-4-4-check (Failed)
        553 - short-H4-RHF-orb-opt-16-1-totenergy (Failed)
        557 - short-H4-orb-opt-1-16-totenergy (Failed)
        558 - short-H4-orb-opt-dmc-16-1 (Failed)
        559 - short-H4-orb-opt-dmc-16-1-totenergy (Failed)
        569 - short-H4-target-descent-16-1-totenergy (Failed)
        571 - short-H4-target-descent-1-16-totenergy (Failed)
        577 - short-H4-batched-hybrid-16-1-totenergy (Failed)
        579 - short-H4-batched-hybrid-1-16-totenergy (Failed)
        596 - short-H4-msd-orb-opt-16-1-totenergy (Failed)
        951 - short-bccH_1x1x1_ae-csvmc-all-nodrift_sdj-16-1-dtotenergy_AB (Failed)
        967 - short-bccH_1x1x1_ae-csvmc-all_sdj-16-1-deeenergy_AB (Failed)
        1092 - short-diamondC_1x1x1_pp-vmcbatch-estimator-onebodydensitymatrices-4-4-check (Failed)
        1098 - short-diamondC_1x1x1_pp-vmc-noJ-estimator-1rdm-4-4-check (Failed)
        1100 - short-diamondC_1x1x1_pp-vmc-J2-estimator-1rdm-4-4-check (Failed)
        1102 - short-diamondC_1x1x1_pp-vmc-estimator-energydensity-cell-4-4-check (Failed)
        1104 - short-diamondC_1x1x1_pp-vmc-estimator-energydensity-voronoi-4-4-check (Failed)
        1106 - short-diamondC_1x1x1_pp-dmc-estimator-energydensity-cell-4-4-check (Failed)
Errors while running CTest
```

**Pull request merged**